templates/demographics_survey_form.html

```diff
@@ -143,7 +135,7 @@ <h1>Demographics and Compensation Settings</h1>
         With the exception of ZIP code, this data may be shared with researchers. We ask for your zip code for internal
         use only; your exact zip code will not be shared with external researchers"><strong>(?)</strong></span>
     </p>
-    <form action="/demographic_survey" onsubmit="return validateForm()" method="POST">
+    <form action="{{'update_demographics'}}" onsubmit="return validateForm()" method="POST">
```

Comment:
I'm unsure what's going on with this syntax. As I understand it 
`{{'update_demographics'}}` would simplify to just `update_demographics`
I would personally prefer we use `url_for` here as well.

Reply:
TIL.

I agree we should go with the longhand here.